app/assets/javascripts/constants/route_constants.es6.js

```diff
@@ -19,6 +19,12 @@ class RoutesSingleton {
       },
     };
   }
+
+  get blueprint() { 
+    return { 
+      home: 'http://www.calblueprint.org/',
+    }
+  };
```

Comment:
Merge remote master into your local branch and fix the merge conflict that you're going to have here!